<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>712</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
background:rgb(69, 69, 69);
color:rgb(220,220,220);
}

QStatusBar{
background:rgb(69, 69, 69);
color:rgb(220,220,220);
}

QLabel{
color:rgb(220,220,220);
}

QAction{
color:rgb(220,220,220);
}

QPushButton{
	color:rgb(220,220,220);
	background:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(48, 48, 48, 255), stop:1 rgba(69, 69, 69, 255));
	border-width: 1px;
	border-color: rgb(177, 214, 49);	
	border-style: solid;
	border-radius: 3px;
	padding: 5px;
}

QGroupBox{
color:rgb(220,220,220);
border-width: 1px;
border-color: rgb(60, 60, 60);
}

QAbstractSpinBox{
color:rgb(220,220,220);
background:rgb(69, 69, 69);
border-width: 1px;
border-color: rgb(177, 214, 49);
border-style: solid;
     border-radius: 3px;
padding: 5px;
}

/* QCombobox Styles*/
QComboBox {
	color:rgb(220,220,220);
    border-style: solid;
	border-width: 1px;
	border-color: rgb(177, 214, 49);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
 }

 QComboBox:editable {
	/* Change the background possibly */
 }

 QComboBox:!editable, QComboBox::drop-down:editable {
	background:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(48, 48, 48, 255), stop:1 rgba(69, 69, 69, 255))
 }

 /* QComboBox gets the &quot;on&quot; state when the popup is open */
 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	background:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(48, 48, 48, 255), stop:1 rgba(69, 69, 69, 255))
 }

 QComboBox:on { /* shift the text when the popup opens */
     padding-top: 3px;
     padding-left: 4px;
 }

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }

 QComboBox::down-arrow:on { /* shift the arrow when popup is open */
     top: 1px;
     left: 1px;
 }

 QComboBox QAbstractItemView {
	color:rgb(220,220,220);
    border-width:1px;
	border-style: solid;
	border-color: rgb(177, 214, 49);
	background: rgba(48, 48, 48, 255);
    selection-background-color:  rgba(69, 69, 69, 255);
 }

/* QGroupBox styles */
 QGroupBox {
     border-width: 1px;
     border-style: solid;
	 border-color: rgb(177, 214, 49);
     border-radius: 3px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
	subcontrol-position: top left;
     subcontrol-origin: margin;
	padding:2 20px;
 }</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>535</width>
     <height>530</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Holoprojector</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="mainView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ViewController" name="viewController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="EncoderController" name="encodeController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="CaptureController" name="captureController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="fileList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>512</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExportSingleFrame"/>
     <addaction name="actionExportEntireVideo"/>
    </widget>
    <addaction name="actionOpenXYZM"/>
    <addaction name="actionOpen_Holovideo"/>
    <addaction name="actionOpen_Holoimage"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConnectCamera"/>
    <addaction name="actionDisconnectCamera"/>
    <addaction name="separator"/>
    <addaction name="actionCalibrateRefrence"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCapture"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRotate"/>
   <addaction name="actionZoom"/>
   <addaction name="actionPan"/>
  </widget>
  <widget class="QToolBar" name="modeBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="modeEncode"/>
   <addaction name="modeCapture"/>
   <addaction name="modeView"/>
  </widget>
  <widget class="QStatusBar" name="infoBar"/>
  <action name="actionOpenXYZM">
   <property name="text">
    <string>Open XYZ</string>
   </property>
  </action>
  <action name="actionOpen_Holovideo">
   <property name="text">
    <string>Open Holovideo</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Holoencoder.qrc">
     <normaloff>:/../../resources/images/Orbit Tool</normaloff>:/../../resources/images/Orbit Tool</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Holoencoder.qrc">
     <normaloff>:/../../resources/images/Zoom Tool</normaloff>:/../../resources/images/Zoom Tool</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>zoom</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Holoencoder.qrc">
     <normaloff>:/../../resources/images/Pan Tool</normaloff>:/../../resources/images/Pan Tool</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="toolTip">
    <string>pan</string>
   </property>
  </action>
  <action name="actionOpen_Holoimage">
   <property name="text">
    <string>Open Holoimage</string>
   </property>
  </action>
  <action name="actionExportSingleFrame">
   <property name="text">
    <string>Single Frame</string>
   </property>
  </action>
  <action name="actionExportEntireVideo">
   <property name="text">
    <string>Entire Video</string>
   </property>
  </action>
  <action name="modeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View Geometry</string>
   </property>
  </action>
  <action name="modeEncode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Encode</string>
   </property>
   <property name="toolTip">
    <string>Encode Geometry</string>
   </property>
  </action>
  <action name="modeCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Capture Geometry</string>
   </property>
  </action>
  <action name="actionCalibrateRefrence">
   <property name="text">
    <string>Calibrate Reference</string>
   </property>
  </action>
  <action name="actionConnectCamera">
   <property name="text">
    <string>Connect Camera</string>
   </property>
  </action>
  <action name="actionDisconnectCamera">
   <property name="text">
    <string>Disconnect Camera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewController</class>
   <extends>QWidget</extends>
   <header>ViewController.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EncoderController</class>
   <extends>QWidget</extends>
   <header>EncoderController.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptureController</class>
   <extends>QWidget</extends>
   <header>CaptureController.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Holoencoder.qrc"/>
 </resources>
 <connections/>
</ui>
